view/index.html

```diff
@@ -1,104 +1,174 @@
-              <button class="article-card-expand-btn"><a href="detail-page-light-sample1.html">continue reading ...</a></button>
+            </button>
           </div>
-          <div class="article-card recent-post-card">
-            <img class="recent-post-img" src="../images/featureImage-Typing2.jpeg" alt="typing on a keyboard">
-              <a href="detail-page-light-sample2.html">
-                <h2>Typing adjustment and how to make it easier</h2>
+        </div>
+        <div class="article-card recent-post-card">
+          <img
+            class="recent-post-img"
+            src="../images/featureImage-Typing2.jpeg"
+            alt="typing on a keyboard"
+          />
+          <div class="recent-post-body">
+            <a href="detail-page-sample3.html">
+              <h2>Typing adjustment and how to make it easier</h2>
+            </a>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            </p>
+            <button class="article-card-expand-btn">
+              <a href="detail-page-sample3.html">
+                continue reading ...
               </a>
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-              <button class="article-card-expand-btn"><a href="detail-page-light-sample2.html">continue reading ...</a></button>
+            </button>
           </div>
-          <a href="browse-page.html">
-            <button class="articles-section-expand-btn">== See More Articles ==</button>
-          </a>
-        </section>
-        <section class="about-author-section">
-            <h2 class="about-author-section-header">
-                Clacks & Thocc
-            </h2>
-            <p class="about-author-section-body">
-                Clacks & Thocc is basically a "Heavily Inspired" online media platform, similar to how China copies every tech giant and sell them for cheap. 
-                We are focused on giving an amatuer, newbie slow push touch to the world of Mechanical Keyboards.
-                That being said, Clacks & Thocc are determined to give professional content to all enthusiast, both new and broke. 
-            </p>
-            <div class="author-card">
-              <img src="../images/user.png" alt="author photo">  
-              <h2 class="author-name">
-                Main Author
-              </h2>
-              <p class="author-description"> 
-                Author is the primary contributor to the site, both content and structure. Author is growing enthusiast for custom builds and modding.
-              </p>
-            </div>
-        </section>
+        </div>
+        <a href="browse-page.html">
+          <button class="articles-section-expand-btn">
+            == See More Articles ==
+          </button>
+        </a>
```

Comment:
[`a`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) and [`button`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) tags should not be a child of any other clickable elements such as `a` and `button` tags.